pygeoapi/openapi.py

```diff
@@ -153,13 +153,14 @@ def get_oas_30(cfg):
             'summary': 'Landing page',
             'description': 'Landing page',
             'tags': ['server'],
+            'operationId': 'getRoot',
```

Comment:
let's align the `operationId` names as much as possible with https://github.com/opengeospatial/ogcapi-features/blob/master/core/examples/openapi/ogcapi-features-1-example1.yaml ?